### ISSUE TITLE
navigator.mediaDevices.getDisplayMedia doesn't works with Edge 17.17134

### ISSUE THREAD

**Issue by lakano** [webrtcHacks/adapter#904]
## Versions affected

**Microsoft Edge 17.17134**

**adapter.js 6.4.5 6.4.6 and future 6.4.7**

## Description
Without adapter.js, we can screen share the desktop/window ( navigator.getDisplayMedia({video:true}) ). If we add adapter.js ( navigator.getDisplayMedia({video:true}) or navigator.mediaDevices.getDisplayMedia({video: true}) ), we can't and receive the error: « invalid calling object » (and no authorization request)

EDIT: For information this works in 6.4.4 without mediaDisplay ( navigator.getDisplayMedia({video:true} ). So from 6.4.5, something break the screen sharing feature.

## Steps to reproduce
I can't share on a jsfiddle because of the iframe. But I've made 2 simple example here:
https://appel.video/issue-edge-with-adapter.html
https://appel.video/issue-edge-without-adapter.html

## Expected results
Permit at least to works as if adapter.js wasn't loaded.

## Actual results
Can't start the screen sharing, with an error.

**Comment by kekkokk** [comment]
Fixed here https://github.com/webrtcHacks/adapter/pull/902

**Comment by lakano** [comment]
Thank you @kekkokk 
Before reporting this issue, I've found your commit on the branch backport-6.4.7, so I've tried to get the release/adapter.js from this branch but without success.
So, I suppose I need to manually generate a new release/adapter.js from this backport-6.4.7 branch, but I don't known how... if you can give me some links/details to help me to get an adapter.js with your fix, this will be really appreciated ! :)

EDIT: I've tried to do a « npm install » and « npm publish » but get theses errors:
```
40 error code ENEEDAUTH
41 error need auth auth required for publishing
42 error need auth You need to authorize this machine using `npm adduser`
```

**Comment by fippo** [comment]
I just published version 6.4.7 to NPM. Sorry for the delay, please holler or reopen if that is still broken

**Comment by fippo** [comment]
*sigh* of course I forgot to pull the 6.4.7 branch from here. 6.4.8 will be up shortly...
